crates/oxc_isolated_declarations/src/scope.rs

```diff
@@ -8,27 +9,28 @@ use oxc_ast::AstBuilder;
 use oxc_ast::{visit::walk::*, Visit};
 use oxc_span::Atom;
 use oxc_syntax::scope::{ScopeFlags, ScopeId};
-use rustc_hash::FxHashSet;
+use rustc_hash::FxHashMap;
+
+bitflags! {
+    #[derive(Debug, Clone, Copy, PartialEq, Eq)]
+    pub struct KindFlags: u8 {
```

Comment:
Could we use `ReferenceFlags`?

Reply:
Yes, but here we don't care about `read` and `write` so add this for understanding easily